odoo/addons/base/models/assetsbundle.py

```diff
@@ -447,6 +463,38 @@ def sanitize(matchobj):
         except CompileError as e:
             return handle_compile_error(e, source=source)
 
+    def run_rtlcss(self, source):
+        rtlcss = 'rtlcss'
+        cmd = [rtlcss, '-']
+
+        # Check the presence of rtlcss, if rtlcss not available then we should return normal less file
+        try:
+            process = Popen(
+                ['rtlcss', '--version'], stdout=PIPE, stderr=PIPE
```

Comment:
as this is only to log the appropriate error, could be done when the compile commands fails